app/importers/file_importers/student_section_assignments_importer.rb

```diff
@@ -14,6 +13,7 @@ def filter
 
   def delete_rows
     StudentSectionAssignment.delete_all
+    ActiveRecord::Base.connection.reset_pk_sequence!('student_section_assignments')
```

Comment:
Can you help me understand this change?  It seems like this would change the core feature of primary keys, where they always refer to the same entity over time.  And so I'm wondering what makes that worth doing here.

I saw your note about not wanting ids to get huge, but doing some napkin math:
- 10k records imported each day
- with 365 days each year
- that's 3.7M records every year
- for 10 years we get 37M records

If I follow then we'd still have two orders of magnitude left until we hit the 4B ceiling in 32-bit integer space.

Reply:
You're right, I'm too sensitive. The number of imported records for the assignments is currently 30,000. But yes, even with a 3 fold increase the id space would still accomodate this for a long time.

This also becomes moot when I change to the isolated deleting scheme I put in the comments anyways.